packages/test/test-end-to-end-tests/src/test/.mocharc.cjs

```diff
@@ -44,11 +44,14 @@ if (runningAgainstInternalRouterliciousCluster) {
 	// https://nodejs.org/api/tls.html#x509-certificate-error-codes).
 	// The flag used to be passed in the npm script in package.json, but adding node-option to the
 	// base mocharc-common.cjs caused it to be ignored, so we need to append it here.
-	if (config["node-option"] === undefined) {
-		config["node-option"] = "use-openssl-ca";
-	} else {
-		config["node-option"] += ",use-openssl-ca";
-	}
+	const baseNodeOptions =
+		config["node-option"] !== undefined
+			? Array.isArray(config["node-option"])
+				? config["node-option"]
+				: [config["node-option"]] // If string, wrap with array
```

Comment:
would it be better to `config["node-option"].split(",")` here? Not sure how mocha ends up plumbing this arg to node.

Reply:
Yeah, good point. I'll merge this since this is the logic we're using in other places and it doesn't seem to have run into issues, but I'll try to double check if something like that might be necessary or at least convenient just in case.